mbuild/tests/test_gmso.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+import pytest
+
+import mbuild as mb
+from mbuild.conversion import from_gmso, to_gmso
+from mbuild.tests.base_test import BaseTest
+from mbuild.utils.io import has_gmso, import_
+
+if has_gmso:
+    gmso = import_("gmso")
+
+
+class TestGMSO(BaseTest):
```

Comment:
I havent tried this, but can you decorate the `class` itself with the `pytest.mark.skipif` decorator?

Reply:
Might save some repeated lines of code